### ISSUE TITLE
Color contrast ratio of selected list item and its background in linked accounts dialog is 1.243:1 which is less than 4.5:1:A11y_SQL Azure Data Studio_Azure Tree item_AI4D

### ISSUE THREAD

**Issue by sspatil123** [microsoft/azuredatastudio#26038]
Please do not close this bug. This bug should only be closed by Trusted Tester after veriﬁcation 
"Check out Accessibility Insights! - Identify accessibility bugs before check-in and make bug ﬁxing faster and easier." 

### GitHub Tags:
#A11yWCAG2.1; #A11ySev2; #A11yTCS; #DesktopApp; #BM_SQL Azure Data Studio_Win32_Nov2024; #SQL Azure Data Studio; #WCAG1.4.3;  #Win32; #Benchmark; #FTP; #AI4D;#ColorContrast;

### Environment Details:
Application Name: SǪL Azure Data Studio 
Edition: Windows 11 Enterprise 
Version: 24H2 
Installed: 22-10-2024 
OS build: 26100.2033 
Experience: Windows Feature Experience Pack 1000.26100.23.0 
Azure data studio version: 1.49.1

### Steps to Reproduce
1.Launch the application-Azure Database Project and install extension SǪL Database Projects. 
2.Tab to “Connections” tab item present in the left menu and Hit Enter Key  
3.Tab to “New connection” button in the connections section and Hit ENTER key. 
4.Tab to “Browse” tab and Hit ENTER key. 
5.Tab to “+” (Azure: sign in) button in the Azure tree item. 
6.“Linked Accounts” dialog will get opened. 
7.Tab to “list items” under Azure section. 
8. Open accessibility insights for windows and check Color contrast ratio for the selected list item and its background. 
9. Observe whether color contrast ratio for the selected list item and its background met the minimum contrast ratio of 4.5:1 or not. 

### Actual Results:
Color contrast ratio of selected list item and its background in linked accounts dialog is 1.243:1 which is less than 4.5:1. 

### Expected Results:
Color contrast ratio of selected list item and its background in linked accounts dialog should be greater than or equal to 4.5:1.  

### 
### User Impact:
Low vision users will not be able to see the radio buttons if the color contrast ratio is less than 4.5:1.

**Comment by kburtram** [comment]
@laurennat could you please look at this compliance issue for March?

**Comment by laurennat** [comment]
@sspatil123 Please test and close this issue. Thank you!

**Comment by sspatil123** [comment]
The issue was fixed hence closing the bug 

![Image](https://github.com/user-attachments/assets/36330f8e-ff87-4c9d-baa5-45bad47d7cc1)
